RELEASENOTES.md

```diff
@@ -196,3 +196,11 @@
   various operations that rely on the `register_view` interface, such as the
   `dev_util.bank_regs` function and the `write-device-reg` and `probe-address`
   CLI commands.
+- `note 6` DMLC no longer emits warnings saying
```

Comment:
Toplevel if logic is 1.4 exclusive, and thus so should this releasenote.

Reply:
I disagree. You are right that they are exclusive for 1.4 *files*, but their only use is in the context of 1.2 *devices*: The only allowed identifier is `dml_1_2`, so the construct is only relevant for common code intended for use in 1.2.

Reply:
>  so the construct is only relevant for common code intended for use in 1.2.
Only for the time being. I think we've both agreed upon extending the conditional to also allow the use of the compat params?

Nevertheless, I guess this is a matter of perspective of where a releasenote belongs. I was thinking "1.4, because this feature is only available in 1.4 files", while you're arguing the use of it is only significant due to how it impact 1.2 devices. Because of that, I'm fine with it being in `RELEASENOTES.dml`; but I would note that your argument sounds like it'd belong in `RELEASENOTES-1.2.md`, which I _would_ heavily disagree with.

Reply:
The releasenote split between 1.2/1.4/both is inherently fuzzy and somewhat questionable. What matters the most is that the end-user sees a note attached to the right package version, this happens no matter what releasenote file we pick; the choice of file is not so important.

I guess the advantage of the split is that many syntax additions are 1.4 specific, and putting it in -1.4 eliminates the need to mention this in text.

I think it would make sense at least to merge RELEASENOTES and RELEASENOTES-1.2. A total 7 notes were 1.2 specific over the past 3 years.